e2e_test/e2e_test.go

```diff
@@ -39,7 +39,32 @@ func TestSendCelo(t *testing.T) {
 	network, shutdown, err := test.NewNetwork(ac, gc, ec)
 	require.NoError(t, err)
 	defer shutdown()
-	ctx, cancel := context.WithTimeout(context.Background(), time.Second*30)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*10)
+	defer cancel()
+
+	accounts := test.Accounts(ac.DeveloperAccounts(), gc.ChainConfig())
+
+	// Send one celo from external account 0 to 1 via node 0.
+	tx, err := accounts[0].SendCelo(ctx, accounts[1].Address, 1, network[0])
+	require.NoError(t, err)
+
+	// Wait for the whole network to process the transaction.
+	err = network.AwaitTransactions(ctx, tx)
+	require.NoError(t, err)
+}
+
+// TestSendCeloV2 starts a network submits a transaction and waits for the whole
+// network to process the transaction.
+func TestSendCeloV2(t *testing.T) {
```

Comment:
Rather than duplicating all the test code, I think we should be able to just run each test twice with slightly different config which I would prefer because it will obviously make things simpler if we end up needing to change any tests.

Reply:
I've managed to set this up, take a look at
* #1950

Reply:
No reason, just didn't want to waste time understanding this to perfection when as soon as this version is out the V2 ones could be deleted and the V2Block would be added as default. But your solution is nice if you are 100% sure this triggers them with the v2 on.

Reply:
Merged it